source/extensions/omni.isaac.lab/omni/isaac/lab/scene/interactive_scene.py

```diff
@@ -155,6 +194,10 @@ def filter_collisions(self, global_prim_paths: list[str] = []):
             global_paths=global_prim_paths,
         )
 
+        # set global prim paths list if not previously defined
+        if len(self._global_prim_paths) < 1:
+            self._global_prim_paths += global_prim_paths
```

Comment:
Should this happen before in the function and we should give self._global_prim_paths to filter collision flags?

I guess this will support the case where people do a mix of using scene and their own spawning (though there's no reason why they should). We should also remove duplicate elements from the list or check for them?

Reply:
If this goes outside the MR scope, let's do it properly in a separate MR like you suggested.

Reply:
Updated.